README.md

```diff
@@ -32,6 +32,10 @@ API.route({
   }
 })
 
+// this is equivalent to creating a create, read, update, and destroy
+// route. Options are folded into every route.
```

Comment:
Hmmm. Better: I should invert the configuration object merging to lay { method: "POST", path: ".." } on top of these options